tests/test_adl_compiler/test_base_compilation.py

```diff
@@ -0,0 +1,65 @@
+import pkg_resources
```

Comment:
pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html